ddp-workspace/projects/ddp-pancan/src/app/components/welcome/welcome.component.scss

```diff
@@ -0,0 +1,391 @@
+@import '../../../styles/variables.scss';
+
+$container-width: 1200px;
+
+.main > * {
+    width: 100%;
+}
+
+section {
+    box-sizing: border-box;
+    text-align: center;
+
+    &:nth-of-type(even) {
+        background-color: #F6F7FA;
+    }
+
+    &.introduction {
+        max-width: 1440px;
+        padding-top: 108px;
+        padding-bottom: 18px;
+        text-align: center;
```

Comment:
guess can be removed since each section has text-align: center; style already

Reply:
Good catch! Fixed at ee510c5723c7990f01eb018fab32b3622424d9a9